### PR TITLE
Feature/refactoring integration into Sulu Standard

### PR DESCRIPTION
**Pull request by dantleech**
Also adds thre PHPCR integrations.

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -116,6 +116,10 @@ sulu_core:
                 snippet: "default"
                 page: "default"
             paths:
+                hnomepage:
```
Comment (danrot) [inline app/config/config.yml]: I guess that's a typo

File: app/config/config.yml [hunk 2]
```diff
@@ -116,6 +116,10 @@ sulu_core:
...
+                    path: %kernel.root_dir%/Resources/pages
+                    internal: false
+                    type: home
                 app_page:
```
Comment (danrot) [inline app/config/config.yml]: Why is this called app_page, and not just page?

Comment (dantleech) [inline app/config/config.yml]: These are meaningless keys. They should be named as you suggeset or changed to `-`. Ultimately this should be refactored as:

````
paths:
    page:
        - /path/to
        - /path/to
    snippet:
        - /path/to
        - /path/to
`````

and we should enable auto discovery for bundles.

Comment (dantleech) [inline app/config/config.yml]: And of course this should be in `sulu_content` not `sulu_core`.

File: app/config/config.yml [hunk 3]
```diff
@@ -116,6 +116,10 @@ sulu_core:
                 snippet: "default"
                 page: "default"
             paths:
+                hnomepage:
+                    path: %kernel.root_dir%/Resources/pages
+                    internal: false
```
Comment (danrot) [inline app/config/config.yml]: Looks like this property has been removed, please adjust it also in this configuration.

**Comment by danrot** [comment]
The ClientWebsiteController needs to be updated as well, Sulu is throwing an exception because of it.

image

**Comment by danrot** [comment]
@dantleech Please be careful with rebasing this branch now, I am also working on it ;-)

**Pull request merged**